### PR TITLE
DOCS-2844: update belfius cbc kbc dotpay eps giropay testing details

### PR DESCRIPTION
**Pull request by Sarah-MultiSafepay**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

**Comment by FrancescaNCook** [comment]
@Sarah-MultiSafepay 

I wonder that a more efficient format for the refunds content would be:

3. Under **Refund**, enter in the:  
    - **Account holder name** field the account holder name of the account you want to refund to. 
    - **IBAN** field the IBAN of the account you want to refund to.
    - **Reason/Description** field the reason for the refund. 
    - **Amount** field the amount to refund.
4. Click **Continue**.
5. Under **Refund confirmation**, check that the description and amount are correct, and then click **Confirm**.
  {{< br >}} A new order is created for the refund. The order status for the refund changes to **Reserved**.
  
  What do you think?

File: content/payments/methods/banks/dotpay/integration-and-testing.md [hunk 1]
```diff
@@ -16,11 +16,51 @@ To process Dotpay payments via our API, see API reference – [Dotpay](/api/#dot
...
-| **Declined** | Transaction was declined |
+Test credentials:
+
+- [API key](/account/site-id-api-key-secure-code/)
+
+**Test a Dotpay order**
+
+1. To test a Dotpay order, send a [redirect](/api/#dotpay) API request.
+2. On the Dotpay page, in the **E-mail address** field, enter any email address.
+3. In the **Phone number** field, enter any phone number.
```
Comment (FrancescaNCook) [inline content/payments/methods/banks/dotpay/integration-and-testing.md:29]:
```suggestion
    - **Phone number** field: Any phone number
```

Comment (FrancescaNCook) [reply content/payments/methods/banks/dotpay/integration-and-testing.md:29]: Update subsequent numbering

File: content/payments/methods/banks/dotpay/integration-and-testing.md [hunk 2]
```diff
@@ -16,11 +16,51 @@ To process Dotpay payments via our API, see API reference – [Dotpay](/api/#dot
...
+4. Select a bank.  
+  You are automatically redirected.
+5. On the Test platform page, from the **Test scenario** list, select **Completed**, and then click **Test**.
+6. The payment is processed in the test environment as **Successful**, with order status **Completed**, and transaction status **Completed**.
```
Comment (FrancescaNCook) [inline content/payments/methods/banks/dotpay/integration-and-testing.md:33]:
```suggestion
    The payment is processed in the test environment as **Successful**, with order status **Completed**, and transaction status **Completed**.
```

File: content/payments/methods/banks/eps/integration-and-testing.md [hunk 3]
```diff
@@ -18,17 +18,62 @@ For the EPS logo, see MultiSafepay GitHub – [MultiSafepay icons](https://githu
 
 {{< details title="View credentials and testing process" >}}
 
-You can test EPS using the Giropay gateway in your MultiSafepay test account. 
+Test credentials:
 
-In your live MultiSafepay account, EPS only appears if you provide the country code for Austria `AT`.
+- [API key](/account/site-id-api-key-secure-code/)
```
Comment (FrancescaNCook) [inline content/payments/methods/banks/eps/integration-and-testing.md:23]:
```suggestion
```

File: content/payments/methods/banks/eps/integration-and-testing.md [hunk 4]
```diff
@@ -18,17 +18,62 @@ For the EPS logo, see MultiSafepay GitHub – [MultiSafepay icons](https://githu
 
 {{< details title="View credentials and testing process" >}}
 
-You can test EPS using the Giropay gateway in your MultiSafepay test account. 
+Test credentials:
```
Comment (FrancescaNCook) [inline content/payments/methods/banks/eps/integration-and-testing.md:21]:
```suggestion
Test credentials: [API key](/account/site-id-api-key-secure-code/)
```

File: content/payments/methods/banks/eps/integration-and-testing.md [hunk 5]
```diff
@@ -18,17 +18,62 @@ For the EPS logo, see MultiSafepay GitHub – [MultiSafepay icons](https://githu
...
 
-Test credentials: You will need a test BIC.
+**Test an EPS order**
 
-Sample statuses:
+1. To test an EPS order, send a [redirect](/api/#eps) API request.
+2. On the EPS page, in the **BIC** field, enter any BIC code. For example, `RZOOAT2L420`.
+3. Click **Confirm**.
+4. On the Test platform page, from the **Test scenario** list, select **Completed**, and then click **Test**.
+5. The payment is processed in the test environment as **Successful**, with order status **Completed**, and transaction status **Completed**.
```
Comment (FrancescaNCook) [inline content/payments/methods/banks/eps/integration-and-testing.md:31]:
```suggestion
    The payment is processed in the test environment as **Successful**, with order status **Completed**, and transaction status **Completed**.
```

**Review by FrancescaNCook — APPROVED** [review]
I've suggested some formatting efficiencies I didn't spot before. See what you think!

File: content/payments/integrations/testing/test-payment-details.md [hunk 6]
```diff
@@ -91,55 +91,253 @@ To test refunding an order:
...
+10. In the **Add transaction comment** field, add a comment, and then click **Add**.
+  The order status changes to **Completed**.
+
+---
+
+**Note:** 
+
+- You can't test cancelling an order. 
+- You can't test sending a refund API request.
+- In the live environment you can't accept a refund order. This is done automatically.
```
Comment (FrancescaNCook) [inline content/payments/integrations/testing/test-payment-details.md:127]:
```suggestion
- In the live environment, you can't accept refund orders, which are done automatically.
```

File: content/payments/integrations/testing/test-payment-details.md [hunk 7]
```diff
@@ -91,55 +91,253 @@ To test refunding an order:
...
```
Comment (FrancescaNCook) [inline content/payments/integrations/testing/test-payment-details.md:166]:
```suggestion
- In the live environment, you can't accept refund orders. These are done automatically.
```

File: content/payments/integrations/testing/test-payment-details.md [hunk 8]
```diff
@@ -91,55 +91,253 @@ To test refunding an order:
...
```
Comment (FrancescaNCook) [inline content/payments/integrations/testing/test-payment-details.md:215]:
```suggestion
- In the live environment, you can't accept refund orders. These are done automatically.
```

File: content/payments/integrations/testing/test-payment-details.md [hunk 9]
```diff
@@ -173,14 +371,40 @@ Sample statuses:
...
```
Comment (FrancescaNCook) [inline content/payments/integrations/testing/test-payment-details.md:407]:
```suggestion
- In the live environment, you can't accept refund orders. These are done automatically.
```

File: content/payments/methods/banks/cbc-kbc/integration-and-testing.md [hunk 10]
```diff
@@ -13,17 +13,45 @@ aliases:
...
+10. In the **Add transaction comment** field, add a comment, and then click **Add**.
+  The order status changes to **Completed**.
+
+---
+
+**Note:** 
+
+- You can't test cancelling an order. 
+- You can't test sending a refund API request.
+- In the live environment you can't accept a refund order. This is done automatically.
```
Comment (FrancescaNCook) [inline content/payments/methods/banks/cbc-kbc/integration-and-testing.md:55]:
```suggestion
- In the live environment, you can't accept refund orders. These are done automatically.
```

File: content/payments/methods/banks/dotpay/integration-and-testing.md [hunk 11]
```diff
@@ -16,11 +16,51 @@ To process Dotpay payments via our API, see API reference – [Dotpay](/api/#dot
...
+10. In the **Add transaction comment** field, add a comment, and then click **Add**.
+  The order status changes to **Completed**.
+
+---
+
+**Note:** 
+
+- You can't test cancelling an order. 
+- You can't test sending a refund API request.
+- In the live environment you can't accept a refund order. This is done automatically.
```
Comment (FrancescaNCook) [inline content/payments/methods/banks/dotpay/integration-and-testing.md:64]:
```suggestion
- In the live environment, you can't accept refund orders. These are done automatically.
```

**Comment by Sarah-MultiSafepay** [comment]
Thanks @FrancescaNCook! Yes, I think that format for the refunds is much cleaner. I've updated the payment methods to follow that format.

**Pull request merged**